sdk/tests/core/test_az_mqtt5_rpc_server_codec.c

```diff
@@ -309,10 +339,11 @@ static void az_mqtt5_rpc_server_codec_parse_received_topic_success(void** state)
 static void az_mqtt5_rpc_server_codec_parse_received_topic_failure(void** state)
```

Comment:
Should this test be renamed? It looks like it's testing generating a subscribe topic successfully now instead of parsing a received topic with a failure

Reply:
Reverted the test back to its original state